adapter/testing/pgxv3.go

```diff
@@ -63,7 +63,9 @@ func testConnDSN(t testing.TB) string {
 	t.Helper()
 
 	testPgConnString, found := os.LookupEnv("TEST_POSTGRES")
-	require.True(t, found, "TEST_POSTGRES env var is not set")
```

Comment:
Most of the tests are running against a DB instance, so I would leave this as it was - skipping 90% of the tests does not make a lot of sense.

Reply:
Well, it makes sense when running `go test -short all`. But there are many packages (including some in standard library) failing e.g. due to missing external dependencies that nobody actually runs `test all` as part of the CI pipeline…